db/migrate/20150806040516_add_timestamps_to_invitations.rb

```diff
@@ -0,0 +1,6 @@
+class AddTimestampsToInvitations < ActiveRecord::Migration
+  def change
+    add_column :invitations, :created_at, :datetime
+    add_column :invitations, :updated_at, :datetime
```

Comment:
You could as well do `add_timestamps :invitations` :) - http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_timestamps